src/test/java/com/teragrep/pth_06/planner/bloomfilter/TableFiltersTest.java

```diff
@@ -69,8 +70,8 @@ class TableFiltersTest {
     // matches IPv4
     final String ipRegex = "(\\b25[0-5]|\\b2[0-4][0-9]|\\b[01]?[0-9][0-9]?)(\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}";
     // matches IPv4 starting with 255.
-    final String ipStartingWith255 = "(\\b25[0-5]?)(\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}";
```

Comment:
why a test case is being changed?

Reply:
This value was not needed by the tests in this test suite but I added the value back

Reply:
please create a ticket to remove in another pr

Reply:
remove the unused value then, no need for it